src/sentry/rules/processing/delayed_processing.py

```diff
@@ -364,39 +365,42 @@ def apply_delayed(project_id: int, *args: Any, **kwargs: Any) -> None:
     now = datetime.now(tz=timezone.utc)
     parsed_rulegroup_to_event_data = parse_rulegroup_to_event_data(rulegroup_to_event_data)
 
-    for rule, group_ids in rules_to_fire.items():
-        frequency = rule.data.get("frequency") or Rule.DEFAULT_FREQUENCY
-        freq_offset = now - timedelta(minutes=frequency)
-        group_to_groupevent = get_group_to_groupevent(
-            parsed_rulegroup_to_event_data, project.id, group_ids
-        )
-        for group, groupevent in group_to_groupevent.items():
-            rule_statuses = bulk_get_rule_status(alert_rules, group, project)
-            status = rule_statuses[rule.id]
-            if status.last_active and status.last_active > freq_offset:
-                logger.info(
-                    "delayed_processing.last_active",
-                    extra={"last_active": status.last_active, "freq_offset": freq_offset},
+    with metrics.timer("delayed_processing.fire_rules.duration"):
```

Comment:
just so i can understand, what's the difference between a timer and a span in these cases? is one better than the other?

Reply:
I think it's just sentry vs datadog